qt/applications/workbench/CMakeLists.txt

```diff
@@ -152,14 +174,31 @@ if(APPLE AND NOT CONDA_BUILD)
     ${CMAKE_CURRENT_SOURCE_DIR}/../../../installers/MacInstaller/Info.plist.in ${CMAKE_CURRENT_BINARY_DIR}/Info.plist
     @ONLY
   )
+  if(CONDA_ENV)
+    set(ENV_VERSION "0.1.0")
```

Comment:
Could I ask for a bit more detail in understanding what this version number relates to?

Reply:
So this version number is required for the `construct.yaml` file as version is a required field in the file. I just had to give it a starting point

Reply:
Ah okay. What would be the reasons that it would need to change?

Reply:
Could you add a small comment around this too?

Other than that I think this is good to go!